### DISCUSSION TITLE
how to run and debug this project using intellij IDEA?

### DISCUSSION THREAD

**Discussion by ivanrein** [apache/netbeans#7220] [Q&A]
I'm trying to run and debug this project using intellij IDEA. I was able to just run by starting `nbbuild.tryme` ant target in IDEA, but running `nbbuild.tryme-debug` failed because of nbjpdastart. 
<img width="809" alt="image" src="https://github.com/apache/netbeans/assets/8987461/bb3788fe-56dc-4d9b-b855-ebdff734a501">

**Comment by matthiasblaesing** [comment]
Why on earth would you use IDEA to debug NetBeans and not NetBeans itself?! With NetBeans you open the module you want to have a closer look at, choose "Debug" from the context menu and the system will start a development version of the IDE in debug mode.

**Reply by ivanrein** [reply]
:) I thought it was kind of a weird question. I have a college assignment to debug this project, but I'm more used to IntelliJ IDEA. I was able to debug using NetBeans the same way you mentioned 👍

**Reply by Chris2011** [reply]
@ivanrein what is missing for you to use NetBeans instead of using IntelliJ to debug NetBeans? Also for what do you need to debug the project and not doing anything for it? Let us know your motivation please :)
